js/sales.js

```diff
@@ -24,56 +24,119 @@
+            allStoresTotal += createStoreTotals(cookieStoreLocations[i], storeTableRow);
         }
+        var totalsRow = createTotalsFooterRow();
+        totalOfAllStoresDaily(totalsRow, allStoresTotal);
     }
 
-    function createStoreHeader(location){
-        var newHeader = document.createElement('h2');
-        newHeader.textContent = location.name;
-        document.body.appendChild(newHeader);
-        return newHeader;
-    }
-
-    function createHoursList(){
-        var newHoursList = document.createElement('ul');
-        document.body.appendChild(newHoursList);
-        return newHoursList;
-    }
-
-    function createHourItems(location, list){
+    function createHoursHeadRow(){
         //use military time
         var openingHour = 6;
         var standardTimeOffSet = 12;
-        for(var i = 0; i < location.simulatedCookiesPerHour.length; i++){
-            var militaryHour =  openingHour + i;
-            var newHourItem = document.createElement('li');
-            var hourSoldCookies = location.simulatedCookiesPerHour[i];
-            if(militaryHour < 12){
+        var hoursOpen = 14;
+        
+        var tableRow = document.createElement('tr');    
+        document.querySelector('thead').appendChild(tableRow);
+        var locationHeader = document.createElement('th');
+        locationHeader.textContent = 'Store Locations';
+        tableRow.appendChild(locationHeader);
+
+        for(var i = 0; i < hoursOpen; i++){
+            var newHourHead = document.createElement('th');
+            newHourHead.setAttribute('scope', 'col');
+            var militaryHour = openingHour + i;
+            if(militaryHour === 12){
+                newHourHead.textContent = '12:00pm';
+            } else if(militaryHour === 24 || militaryHour === 0){
+                newHourHead.textContent = '12:00am';
+            } else if(militaryHour < 12){
                 //1am-11am
-                newHourItem.textContent = militaryHour + 'am: ' + hourSoldCookies + " cookies";
-            } else if(militaryHour === 12){
-                newHourItem.textContent = '12pm: ' + hourSoldCookies + ' cookies';
-            } else if(militaryHour === 24){
-                newHourItem.textContent = '12am: ' + hourSoldCookies + ' cookies';
+                newHourHead.textContent = militaryHour + ':00am';
             } else {
                 //1pm-11pm
-                newHourItem.textContent = (militaryHour - standardTimeOffSet) + 'pm: ' + hourSoldCookies + ' cookies';
+                newHourHead.textContent = (militaryHour - standardTimeOffSet) + ':00pm';
+            }
+            tableRow.appendChild(newHourHead);
+        }
+        return tableRow;
+    }
+
+    CookieStore.prototype.render = function(){
+        var tableStoreRow = document.createElement('tr');
+        tableStoreRow.setAttribute('scope', 'row');
+        var tableStoreHead = document.createElement('th');
+        tableStoreHead.textContent = this.name;
```

Comment:
Good job setting the first non-empty cell of the row to a `<th>` element! We should move the `scope="row"` assignment to this `<th>` instead of on the `<tr>`, but I _constantly_ see developers choose not to create the first cell separately from the others because "it's too much work" to have one `<th>` and everything else be a `<td>`. Really happy to see you taking the time to do this (especially since it's actually pretty straightforward).